js/Storages/ProjectStorage.js

```diff
@@ -23,10 +23,17 @@ SPM.Storages.ProjectStorage = {
 		return projects;
 	},
 
-	getMyProjects: function (projects) {
-		return (this.myProjects)?
-			Promise.resolve(this.myProjects.map(function (projectId) { return this.projects[projectId] }.bind(this))):
-			Promise.reject("No data")
+	getProjectsByUser: function (user) {
```

Comment:
Okay, you obviously did a refacto here, but I fail to see the reason, as `getProjectsByUser` is only called with `members.me`. Besides, I think it might break something : 
Suppose we call `getProjectByName`, which returns a project whose I am a member. The project is saved in the storage.
Then we call `findProjectByUser(members.me)`
It returns a project I belong to, but not all of them.